mindsdb/integrations/handlers/litellm_handler/settings.py

```diff
@@ -6,9 +6,8 @@ class CompletionParameters(BaseModel):
     model: str  # The model to be used for the API call.
     prompt_template: Optional[str] = None  # Template for the prompt to be used.
 
-    # Optional OpenAI params: see https://platform.openai.com/docs/api-reference/chat/create
-    functions: List = []  # List of functions for advanced model operations.
-    function_call: str = ""  # String to specify a particular function call, if needed.
+    # Optional OpenAI params: see  https://platform.openai.com/docs/api-reference/runs
```

Comment:
This link refers to the Assistants API, but I think we are calling the Chat Completions API within the handler. Should it be changed?